package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "kuzzle",
   "author": "The Kuzzle Team <support@kuzzle.io>",
-  "version": "1.6.2",
+  "version": "1.6.5",
```

Comment:
???

Reply:
I didn't know why there was the wrong version in the package.json, I updated it to the next release version (I wanted to do a hotfix at first)

Reply:
I merged master onto 1-dev, so we now have the correct version number on the development branch.
But you still need to fix that one.

Reply:
Fixed